dist/index.js

```diff
@@ -55,29 +55,20 @@ function run() {
     });
 }
 exports.run = run;
-function getLatestRTXAssetURL() {
```

Comment:
I really dislike committing compiled code into the repo, but that's what the action template does so I assume it's required?